db/data/queries/createDatabase.sql

```diff
@@ -1 +1,2 @@
 CREATE DATABASE `quickOrder`;
+ALTER DATABASE `quickOrder` COLLATE = `utf8_general_ci`;
```

Comment:
tried that already, actually it's the first line in `createTable.sql`
i think i just need to change something in the db settings